src/queue/bookmark/activities.ts

```diff
@@ -0,0 +1,32 @@
+import { InjectedProps } from '../common';
+import { triggerTypedEvent } from '../../common';
+import { Bookmark } from '../../entity';
+
+interface CommonBookmarkReminderParams {
+  userId: string;
+  postId: string;
+}
+
+export const createActivities = ({ con, logger }: InjectedProps) => ({
```

Comment:
You can import global logger instance, no need to pass it through arguments.

Reply:
Cool. Let's do that.